apps/opik-frontend/src/components/pages/PlaygroundPage/PlaygroundOutputs/PlaygroundOutputScores/PlaygroundOutputScoresContainer.tsx

```diff
@@ -0,0 +1,148 @@
+import React, { useMemo, useRef, useEffect } from "react";
+
+import useTraceById from "@/api/traces/useTraceById";
+import useRulesList from "@/api/automations/useRulesList";
+import useProjectByName from "@/api/projects/useProjectByName";
+import useAppStore from "@/store/AppStore";
+import { PLAYGROUND_PROJECT_NAME } from "@/constants/shared";
+import { getScoreNamesFromRule } from "@/lib/rules";
+import PlaygroundOutputScores, { ScoreData } from "./PlaygroundOutputScores";
+
+const REFETCH_INTERVAL = 5000;
+const MAX_REFETCH_TIME = 300000;
+
+interface PlaygroundOutputScoresContainerProps {
+  traceId: string | null;
+  selectedRuleIds: string[] | null | undefined;
+  stale?: boolean;
+  className?: string;
+}
+
+const PlaygroundOutputScoresContainer: React.FC<
+  PlaygroundOutputScoresContainerProps
+> = ({ traceId, selectedRuleIds, stale = false, className }) => {
+  const workspaceName = useAppStore((state) => state.activeWorkspaceName);
+  const pollingStartTimeRef = useRef<number | null>(null);
+
+  useEffect(() => {
+    pollingStartTimeRef.current = traceId ? Date.now() : null;
+  }, [traceId]);
+
+  // Check if we should show metrics AFTER early return check
+  const shouldShowMetrics =
+    selectedRuleIds !== undefined &&
+    !(Array.isArray(selectedRuleIds) && selectedRuleIds.length === 0);
```

Comment:
```suggestion
const shouldShowMetrics = Array.isArray(selectedRuleIds) && selectedRuleIds.length > 0;
```

Array.isArray should return false, for undefined